docs/next/guides/integrate-with-react/react-simple-example.md

```diff
@@ -26,7 +26,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     
-    this.handsontableData = Handsontable.helper.createSpreadsheetData(100, 50);
+    this.handsontableData = Handsontable.helper.createSpreadsheetData(6, 10);
```

Comment:
Because we have such behavior in the current documentation https://handsontable.com/docs/8.3.2/frameworks-wrapper-for-react-simple-examples.html and these large amounts of data don't show specific use-case.